### ISSUE TITLE
Chart colours meeting WCAG

### ISSUE THREAD

**Issue by rogercherry** [NHSDigital/data-viz-community-of-practice#1]
Hi,

I had a question about your chart colours.
I noticed that the second blue #41B6E6 doesn’t meet the minimum contrast ratio against white to pass WCAG AA standard (https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). 
Do we not have to meet this as a minimum for all NHS products? If so should the colour palette be modified to make sure this is met?

**Comment by magnusMaximus** [comment]
Roger Cherry
I am curious. How did you work that out? :-)

**Comment by rogercherry** [comment]
Put it into this colour contrast checker
https://webaim.org/resources/contrastchecker/

**Comment by SimonMunnsNHSE** [comment]
Hu that's simple.

**Comment by rogercherry** [comment]
Just prompting this in case anything that needs to be changed to make sure these colours pass WCAG AA standard

**Comment by davidhardy5-nhs** [comment]
This has been noted and will be picked up as soon as the new data viz accessible colours are published thanks!
